util/chplenv/third-party-pkgs

```diff
@@ -40,18 +39,16 @@ def get_all_pkgs():
 def find_pkg_aliases():
     """ Find any aliases for third-party packages """
     pkg_aliases = dict()
-    for pkg in get_all_pkgs():
-        make_include = os.path.join(get_chpl_third_party(), pkg, 'Makefile.include')
-        alias_var = '{0}_ALIASES'.format(pkg.upper())
-        if os.path.isfile(make_include):
-            with open(make_include, 'r') as f:
-                for line in f.readlines():
-                    var_value = line.strip().split('=')
-                    if var_value[0] == alias_var:
-                        aliases = var_value[1].split()
-                        for alias in aliases:
-                            pkg_aliases[alias] = pkg
-                        break
+
+    pkg_aliases['CHPL_LLVM-clang-included'] = 'llvm'
+    pkg_aliases['CHPL_TASKS-qthreads'] = 'qthread'
+    pkg_aliases['CHPL_GMP-bundled'] = 'gmp'
+    pkg_aliases['CHPL_HWLOC-bundled'] = 'hwloc'
+    pkg_aliases['CHPL_JEMALLOC-bundled'] = 'jemalloc'
+    pkg_aliases['CHPL_LIBFABRIC-bundled'] = 'libfabric'
+    pkg_aliases['CHPL_LLVM-bundled'] = 'llvm'
+    pkg_aliases['CHPL_UNWIND-bundled'] = 'libunwind'
```

Comment:
Due to the way this is working now - I wouldn't think of it as a dictionary of aliases at all - but rather a mapping from settings to which third-party subdir to build.  So maybe we could call the function `generate_subdir_map` or something like that.

Reply:
I would probably have made a map of maps, e.g. `map['CHPL_LLVM']['clang-included'] = 'llvm'`. I think that's slightly cleaner but what you have here is OK with me.

Reply:
I opened https://github.com/chapel-lang/chapel/issues/16768 about following up with your other suggestions.